AddOns/BasicFinancials/app/app.json

```diff
@@ -0,0 +1,40 @@
+{
+  "id": "0242c240-2336-450f-9678-2f13f4ce9a6e",
+  "name": "Basic Financials",
+  "publisher": "Microsoft",
+  "brief":  "Basic Financial Extensions aligns the User Experience to the license limitations",
+  "description": "This extension aligns the User Experience to the Basic Financial license limitations, by disabling certain Essential Experience Application Areas And by creating two new Experience Application Areas 'Basic Financials' and 'Orders'. The 'Basic Financials' Application Area is used to enable Layout and Actions, where the 'Orders' Application Area is used to disable Layout and Actions.",
```

Comment:
This is probably more technical than what should be in a description. I suggest something more like the following:

The Basic Financials extension gives small companies access to the Business Central features that are at the absolute core of doing business. For example, when you install this extension you can invoice customers, pay vendors, manage inventory, and keep your finances in order.